Application/Source/Detail/View/DetailViewController.swift

```diff
@@ -81,4 +81,8 @@ extension DetailViewControllerFactoryProtocol {
         return DetailViewController(viewModel: viewModel, themeProvider: themeProvider)
     }
 
+    func makeFoodInfoViewController(viewModel: FoodInfoViewModel) -> FoodInfoViewController {
```

Comment:
I don't believe this is needed. This function should be implemented by the protocol extension.